packages/react/src/components/Forms/LabelledBox/index.stories.tsx

```diff
@@ -7,17 +7,43 @@ export default {
   component: LabelledBox,
 } as Meta<typeof LabelledBox>;
 
+const DebugContainer = ({ children }: React.PropsWithChildren<{}>) => {
+  const style: React.CSSProperties = {
+    height: "3.5rem",
+    // A grid background
+    backgroundImage: `linear-gradient(to right, #eee 1px, transparent 1px), linear-gradient(to bottom, #eee 1px, transparent 1px)`,
+    border: "thin solid #eee",
```

Comment:
Just to prevent bolder border on top and left sides.

```suggestion
    border: "solid #eee",
    borderWidth: '0 1px 1px 0',
```